GoodDollar/GoodDAPP
# (Feature)  #694 eToro test add marketplace icon to dashboard

# Description

displays the icon if "MARKET" env variable is set. The default value is true if the network is "etoro".
On click opens in frame emarket.gooddollar.org site.

Fixes #694 

# How Has This Been Tested?

1) Set REACT_APP_MARKET=true or set default value (true) for REACT_APP_MARKET in config file.
2) Register/login to the app
3) See the Marketplace icon at the top of the dashboard page.

# Checklist:

- Gif files:
REACT_APP_MARKET env set to true:
![2019-10-11_14-13-24 (1)](https://user-images.githubusercontent.com/49862004/66648109-672cfe80-ec33-11e9-942a-0121ff2bcdc0.gif)
REACT_APP_MARKET env set to fasle:
![2019-10-11_14-12-32 (1)](https://user-images.githubusercontent.com/49862004/66648112-68f6c200-ec33-11e9-9c86-2c94330995f5.gif)

- new env variable:
REACT_APP_MARKET - optional
